diff --git a/build/src/utils/npm.js b/build/src/utils/npm.js
index b861322e982f936bd3267c507a56f48213cc3d16..c1116bc1dbf2f5cce01f821a28a1ebbe2eeb7849 100644
--- a/build/src/utils/npm.js
+++ b/build/src/utils/npm.js
@@ -170,7 +170,8 @@ async function extractNpmTarballAsync(stream, props) {
     transformStream.on("data", (chunk)=>{
         hash.update(chunk);
     });
-    await pipeline(stream, transformStream, _tar().default.extract({
+    const tar = _tar().default ?? _tar();
+    await pipeline(stream, transformStream, tar.extract({
         cwd,
         filter,
         onentry: (0, _createFileTransform.createEntryResolver)(name),
