diff --git a/dist/types/config-object.type.js b/dist/types/config-object.type.js
new file mode 100644
index 0000000000000000000000000000000000000000..c5f6fa2387d71c5c7843256b35e6a97961a95e82
--- /dev/null
+++ b/dist/types/config-object.type.js
@@ -0,0 +1,3 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+
diff --git a/dist/types/config.type.js b/dist/types/config.type.js
new file mode 100644
index 0000000000000000000000000000000000000000..c5f6fa2387d71c5c7843256b35e6a97961a95e82
--- /dev/null
+++ b/dist/types/config.type.js
@@ -0,0 +1,3 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+
diff --git a/dist/types/no-infer.type.js b/dist/types/no-infer.type.js
new file mode 100644
index 0000000000000000000000000000000000000000..c5f6fa2387d71c5c7843256b35e6a97961a95e82
--- /dev/null
+++ b/dist/types/no-infer.type.js
@@ -0,0 +1,3 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+
diff --git a/dist/types/path-value.type.js b/dist/types/path-value.type.js
new file mode 100644
index 0000000000000000000000000000000000000000..c5f6fa2387d71c5c7843256b35e6a97961a95e82
--- /dev/null
+++ b/dist/types/path-value.type.js
@@ -0,0 +1,3 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+
